R/col_types.R

```diff
@@ -364,8 +364,27 @@ col_spec_standardise <- function(file, col_names = TRUE, col_types = NULL,
   spec
 }
 
-guess_types <- function(datasource, tokenizer, locale, n = 1000) {
-  guess_types_(datasource, tokenizer, locale, n = n)
+
+check_guess_max <- function(guess_max, max_limit = .Machine$integer.max %/% 100) {
+  if (!(length(guess_max) == 1 && is.numeric(guess_max) &&
```

Comment:
Hmmmmm, I think we can just stick to an `if` statement instead of introducing a new syntactic form 😉